src/runtime/Makefile

```diff
@@ -125,6 +126,7 @@ TDVFFIRMWAREPATH := $(PREFIXDEPS)/share/tdvf/OVMF_CODE.fd
 TDVFFIRMWAREVOLUMEPATH := $(PREFIXDEPS)/share/tdvf/OVMF_VARS.fd
 TDSHIMFIRMWAREPATH := ${PREFIXDEPS}/share/td-shim/td-shim.bin
 SEVFIRMWAREPATH := $(PREFIXDEPS)/share/ovmf/OVMF.fd
+SNPFIRMWAREPATH := $(PREFIXDEPS)/share/ovmfx86_64/OVMF.fd
```

Comment:
I'm curious where this location comes from? At least worth a comment.

Joana's guide has:

```
firmware = "/path/to/kata-containers/tools/packaging/static-build/ovmf/opt/kata/share/ovmf/OVMF.fd"
```

Reply:
Based on my ovmf changes here: https://github.com/kata-containers/kata-containers/pull/5850

My reasoning to have it different than /share/ovmf/ is that it would overwrite the SEV build, and we will want both in the cc bundle at the same time.

Reply:
OK